src/actions/index.ts

```diff
@@ -15,7 +14,7 @@ export const RECEIVE_RECORDING_INFO = 'RECEIVE_RECORDING_INFO'
 
 export const ADD_RECORDING = 'ADD_RECORDING'
 export const DELETE_RECORDINGS = 'DELETE_RECORDINGS'
-export const SET_RECORDING_INFO = 'SET_RECORDING_INFO'
```

Comment:
We know longer set recording info on the redux state. recordingInfo and sortingInfo are not stored in the redux state. This is a simplification -- the prior system was awkward.